apps/nfid-frontend-e2e/src/steps/when.ts

```diff
@@ -107,3 +107,43 @@ When(/^User clicks the back button in Send window$/, async () => {
     Assets.switchSendType,
   )
 })
+
+When(/^User clicks the (.*) option button$/, async (option: string) => {
+  await Assets.getTokenOption(option).click()
+  await browser.pause(10000)
+})
+
+When(/^User clicks the ShowHide button of (.*) token$/, async (tokenName: string) => {
+  await Assets.ManageTokensDialog.tokenShowHideButton(tokenName).then(async (it) => {
+    await it.waitForDisplayed()
+    await browser.pause(1500)
+    await it.click()
+    await browser.pause(10000)
```

Comment:
this pause is needed as hide/show process takes 8-10 sec and there is no any element to use waitUntil